email_alert_service/models/document_links.rb

```diff
@@ -0,0 +1,53 @@
+class DocumentLinks
+  attr_reader :document
+
+  def initialize(document)
+    @document = document
+  end
+
+  def self.call(args)
+    new(args).links
```

Comment:
```suggestion
  def self.call(...)
    new(...).links
```

Minor, you can use the new argument forwarding operator: https://blog.saeloun.com/2019/12/04/ruby-2-7-adds-new-operator-for-arguments-forwarding.html